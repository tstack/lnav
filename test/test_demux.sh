#! /bin/bash

export YES_COLOR=1
export TZ=UTC

cat ${test_dir}/logfile_docker_compose.0 | run_cap_test env TEST_COMMENT="docker-demux-no-ts" \
     ${lnav_test} -n

cat ${test_dir}/logfile_docker_compose_with_ts.0 | run_cap_test env TEST_COMMENT="docker-demux-with-ts" \
     ${lnav_test} -n

run_cap_test ${lnav_test} -n ${test_dir}/logfile_docker_compose_with_ts.0

run_cap_test ${lnav_test} -n ${test_dir}/logfile_mux_zookeeper.0

run_cap_test ${lnav_test} -n \
    -c ';SELECT * FROM lnav_file_demux_metadata' \
    ${test_dir}/logfile_mux_zookeeper.0

run_cap_test ${lnav_test} -n ${test_dir}/logfile_docker_compose_with_noise.0

run_cap_test ${lnav_test} -n \
    -c ':switch-to-view text' \
    ${test_dir}/logfile_docker_compose_with_noise.0
