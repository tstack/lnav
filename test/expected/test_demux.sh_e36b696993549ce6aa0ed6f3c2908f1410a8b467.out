[31mout of frame line 1[0m
[31mout of frame line 2[0m
